ui/compontents.go

```diff
@@ -65,7 +65,7 @@ func (page pageCommon) layoutUSDBalance(gtx layout.Context) layout.Dimensions {
 					return border.Layout(gtx, func(gtx C) D {
 						return padding.Layout(gtx, func(gtx C) D {
 							amountDCRtoUSDString := formatUSDBalance(page.printer, page.amountDCRtoUSD)
-							return page.theme.Label(values.TextSize14, amountDCRtoUSDString).Layout(gtx)
+							return page.theme.LabelColor(values.TextSize14, amountDCRtoUSDString, page.theme.Color.DeepBlue).Layout(gtx)
```

Comment:
Instead of using Label, you code can best be...
```suggestion
							 return page.theme.Body2(amountDCRtoUSDString).Layout(gtx)
```

Also, you dont need the LabelColor() method.